disabled_appveyor.yml

```diff
@@ -0,0 +1,81 @@
+# This file was automatically generated by conda-smithy. To update a component of this
```

Comment:
It seems a bit odd that it generates this anyway and names it `disabled-` - I assume that's just how conda-smithy works?